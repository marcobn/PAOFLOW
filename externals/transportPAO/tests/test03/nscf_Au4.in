Infinite Gold Chain
4 atom/cell, Lattice Parameter = 2.5 Ang
   &CONTROL
      calculation = 'nscf',
      restart_mode='from_scratch',
      verbosity='default',
      prefix='gold4',
      nstep=100,
      tstress = .false.,
      tprnfor = .true.,
      pseudo_dir = '../pseudo',
      outdir = './SCRATCH/'
      wf_collect = .TRUE.
   /
   &SYSTEM
      ibrav=8, 
      celldm(1)=20.0, 
      celldm(2)=1.0,
      celldm(3)=.944864
      nat=4, 
      ntyp=1,
      ecutwfc=50.0,
      ecutrho=200.0,
      occupations='smearing',
      smearing='gaussian',
      degauss=0.01,
      nbnd = 40
      noinv = .TRUE.
      nosym = .TRUE.
   /
   &ELECTRONS
      electron_maxstep = 100,
      mixing_mode = 'plain',
      mixing_beta = 0.3,
      conv_thr =  1.0d-8,
   /
ATOMIC_SPECIES
  Au   196.9665   Au11pw91.mt.UPF
ATOMIC_POSITIONS {angstrom}
Au   0.0  0.0 0.0
Au   0.0  0.0 2.5
Au   0.0  0.0 5.0
Au   0.0  0.0 7.5
K_POINTS {automatic}
1 1 4  0 0 0

