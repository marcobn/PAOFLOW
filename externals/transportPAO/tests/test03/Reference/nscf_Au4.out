
     Program PWSCF v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:20:20 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Reading input from /home/ferretti/want/tests/test16/nscf_Au4.in

     Atomic positions and unit cell read from directory:
     ./SCRATCH/gold4.save/
 

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

     Found symmetry operation: I + (  0.0000  0.0000  0.2500)
     This is a supercell, fractional translations are disabled
 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        1060    1060    277                60174    60174    8054
     Max        1062    1062    279                60175    60175    8055
     Sum        6369    6369   1669               361045   361045   48327
 


     bravais-lattice index     =            8
     lattice parameter (alat)  =      20.0000  a.u.
     unit-cell volume          =    7558.9120 (a.u.)^3
     number of atoms/cell      =            4
     number of atomic types    =            1
     number of electrons       =        44.00
     number of Kohn-Sham states=           40
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     200.0000  Ry
     Exchange-correlation      =  SLA  PW   GGX  GGC ( 1 4 2 2 0)
     EXX-fraction              =        0.00

     celldm(1)=  20.000000  celldm(2)=   1.000000  celldm(3)=   0.944864
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   0.944864 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.058353 )  


     PseudoPot. # 1 for Au read from file:
     ../pseudo/Au11pw91.mt.UPF
     MD5 check sum: e270e13d0e45fa10676aee720c7982d9
     Pseudo is Norm-conserving + core correction, Zval = 11.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  331 points,  2 beta functions with: 
                l(1) =   1
                l(2) =   2

     atomic species   valence    mass     pseudopotential
        Au            11.00   196.96650     Au( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Au  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Au  tau(   2) = (   0.0000000   0.0000000   0.2362158  )
         3           Au  tau(   3) = (   0.0000000   0.0000000   0.4724315  )
         4           Au  tau(   4) = (   0.0000000   0.0000000   0.7086473  )

     number of k points=     4  gaussian smearing, width (Ry)=  0.0100
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.5000000
        k(    2) = (   0.0000000   0.0000000   0.2645883), wk =   0.5000000
        k(    3) = (   0.0000000   0.0000000  -0.5291767), wk =   0.5000000
        k(    4) = (   0.0000000   0.0000000  -0.2645883), wk =   0.5000000

     Dense  grid:   361045 G-vectors     FFT dimensions: (  96,  96,  90)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         4.59 Mb     (   7527,   40)
        NL pseudopotentials             3.68 Mb     (   7527,   32)
        Each V/rho on FFT grid          2.11 Mb     ( 138240)
        Each G-vector array             0.46 Mb     (  60174)
        G-vector shells                 0.11 Mb     (  14938)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions        18.38 Mb     (   7527,  160)
        Each subspace H/S matrix        0.39 Mb     ( 160, 160)
        Each <psi_i|beta_j> matrix      0.02 Mb     (     32,   40)

     Check: negative/imaginary core charge=   -0.000021    0.000000

     The potential is recalculated from file :
     ./SCRATCH/gold4.save/charge-density.dat

     Starting wfc are   36 randomized atomic wfcs

     total cpu time spent up to now is        0.9 secs

     per-process dynamical memory:    39.4 Mb

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  2.27E-11,  avg # of iterations = 50.0

     total cpu time spent up to now is       66.1 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000     band energies (ev):

   -10.8561  -8.5238  -8.5238  -7.8676  -7.8672  -7.1979  -7.1979  -7.1978
    -7.1978  -6.8048  -6.8047  -6.4774  -6.4773  -6.4773  -6.4772  -6.4532
    -6.1291  -6.1289  -5.4937  -5.4934  -5.3437  -5.2259  -5.2259  -1.1775
    -1.1775  -0.2743   1.0391   1.0391   1.0418   1.0418   1.1483   2.0732
     2.1168   2.6205   2.6234   2.8332   2.8332   2.8941   2.9962   3.5876

          k = 0.0000 0.0000 0.2646     band energies (ev):

   -10.6528  -9.0948  -8.4340  -8.4340  -7.7478  -7.7478  -6.7805  -6.7804
    -6.6288  -6.6050  -6.6049  -6.5683  -6.5683  -6.3949  -6.3467  -6.3466
    -6.1564  -6.1562  -5.9301  -5.4563  -5.4300  -5.4300  -4.7454  -1.5550
    -0.9820  -0.9820   0.2146   0.2146   1.2403   1.9767   1.9846   1.9846
     2.2090   2.9351   2.9351   2.9461   2.9853   3.0856   3.4271   3.4542

          k = 0.0000 0.0000-0.5292     band energies (ev):

   -10.0536 -10.0533  -8.1700  -8.1700  -8.1697  -8.1697  -6.7110  -6.7109
    -6.7108  -6.7107  -6.2336  -6.2334  -6.2333  -6.2332  -6.2210  -6.2208
    -5.9369  -5.9369  -5.9367  -5.9367  -5.8283  -5.8281  -3.3639  -3.3638
    -0.4780  -0.4780  -0.4780  -0.4780   1.5163   1.5163   2.4854   2.4854
     3.0328   3.0328   3.0328   3.0328   3.2374   3.2374   3.2374   3.2374

          k = 0.0000 0.0000-0.2646     band energies (ev):

   -10.6528  -9.0948  -8.4340  -8.4340  -7.7478  -7.7478  -6.7805  -6.7804
    -6.6288  -6.6050  -6.6049  -6.5683  -6.5683  -6.3949  -6.3467  -6.3466
    -6.1564  -6.1562  -5.9301  -5.4563  -5.4300  -5.4300  -4.7454  -1.5550
    -0.9820  -0.9820   0.2146   0.2146   1.2403   1.9767   1.9846   1.9846
     2.2090   2.9351   2.9351   2.9461   2.9853   3.0856   3.4271   3.4542

     the Fermi energy is    -5.2092 ev

     Writing output data file gold4.save
 
     init_run     :      0.74s CPU      0.78s WALL (       1 calls)
     electrons    :     64.34s CPU     65.20s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.01s CPU      0.00s WALL (       1 calls)
     potinit      :      0.30s CPU      0.33s WALL (       1 calls)

     Called by electrons:
     c_bands      :     64.34s CPU     65.20s WALL (       1 calls)
     v_of_rho     :      0.22s CPU      0.25s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.02s WALL (       4 calls)
     cegterg      :     61.50s CPU     61.94s WALL (      12 calls)

     Called by *egterg:
     h_psi        :     47.74s CPU     47.93s WALL (     216 calls)
     g_psi        :      0.52s CPU      0.54s WALL (     200 calls)
     cdiaghg      :      1.89s CPU      1.90s WALL (     204 calls)

     Called by h_psi:
     add_vuspsi   :      1.04s CPU      1.03s WALL (     216 calls)

     General routines
     calbec       :      1.21s CPU      1.19s WALL (     216 calls)
     fft          :      0.16s CPU      0.17s WALL (      11 calls)
     fftw         :     39.39s CPU     39.66s WALL (    6286 calls)
     davcio       :      0.01s CPU      0.42s WALL (       8 calls)
 
     Parallel routines
     fft_scatter  :     14.92s CPU     14.83s WALL (    6297 calls)
 
     PWSCF        :  1m 6.02s CPU     1m 7.23s WALL

 
   This run was terminated on:  20:21:27  17Apr2014            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
